src/NetMQ/NetMQActor.cs

```diff
@@ -229,17 +229,11 @@ private void RunShim()
                 m_shimHandler.Run(m_shim);
             }
             catch (TerminatingException)
-            {}
-
-            // Do not block, if the other end of the pipe is already deleted
-            m_shim.Options.SendTimeout = TimeSpan.Zero;
-
-            try
             {
-                m_shim.SignalOK();
             }
-            catch (AgainException)
-            {}
```

Comment:
Should `AgainException` be marked obsolete? Client code may be updated to use the new API, yet still have `catch (AgainException)` blocks.